### PR TITLE
[AUD-1407] Add stylelint

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Adds prettier linting to our css. also includes css modules plugin to handle `composes` key correctly

### Dragons

App looks the same from what i can tell, linting does a pretty good job at preserving accuracy. We should do a few smoke tests to confirm

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-add-stylelint

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-add-stylelint

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-add-stylelint

File: package.json [hunk 1]
```diff
@@ -131,6 +131,8 @@
     "eject": "react-scripts eject",
     "lint:fix": "eslint --cache --fix --ext=js,jsx,ts,tsx src",
     "lint": "eslint --cache --ext=js,jsx,ts,tsx src",
+    "stylelint": "stylelint 'src/**/*.css'",
```
Comment (raymondjacobson) [inline package.json:134]: I wonder if it's worth merging stylelint into the same script as lint, e.g. elslint && styelint

**Review by raymondjacobson — APPROVED** [review]
LGTM! I played around w. the demo site for a bit, didn't see anything jump out at me.

**Pull request merged**